src/main/java/uk/gov/hmcts/reform/sandl/snlevents/scheduledtasks/UserTransactionsRollbackScheduledTask.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.sandl.snlevents.scheduledtasks;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sandl.snlevents.model.db.UserTransaction;
+import uk.gov.hmcts.reform.sandl.snlevents.service.UserTransactionService;
+
+import java.util.List;
+import javax.transaction.Transactional;
+
+@Component
+@Transactional
+@EnableScheduling
+@ConditionalOnProperty(
+    prefix = "scheduler.auto-rollback",
+    name = "enabled", havingValue = "true")
+public class UserTransactionsRollbackScheduledTask {
+    private static final Logger logger = LoggerFactory.getLogger(UserTransactionsRollbackScheduledTask.class);
```

Comment:
Just a suggestion, there is @ Log4j annotation im lombok

Reply:
@Slf4j to be precise, but 👍